packages/voice/src/rest/v1/applications/applications-api.ts

```diff
@@ -62,7 +62,7 @@ export class ApplicationsApi extends VoiceApi {
     };
 
     const body: RequestBody = '';
-    const path = `/v1/calling/query/number/${data['number']}`;
+    const path = `/v1/calling/query/number/${data['number'].split(" ").join("")}`;
```

Comment:
May be a comment about current server behavior explaining 'why' we have to modify input with such "trick".
For future generation education purpose or even ourselves in few weeks/months 🥲

Reply:
I've added a comment